uvicorn/config.py

```diff
@@ -101,9 +101,18 @@
 logger = logging.getLogger("uvicorn.error")
 
 
-def create_ssl_context(certfile, keyfile, ssl_version, cert_reqs, ca_certs, ciphers):
+def create_ssl_context(
+    certfile, keyfile, password, ssl_version, cert_reqs, ca_certs, ciphers
+):
     ctx = ssl.SSLContext(ssl_version)
-    ctx.load_cert_chain(certfile, keyfile)
+    if password:
+
+        def getpassword():
+            return password
+
+        ctx.load_cert_chain(certfile, keyfile, getpassword)
+    else:
+        ctx.load_cert_chain(certfile, keyfile)
```

Comment:
(Nit) Dunno if we could go for a more compact form - hopefully this is readable enough? Or can we _not_ pass `None` as the `password` argument? (The Python docs aren't very explicit about that.)

```suggestion
    password = (lambda: password) if password else None
    ctx.load_cert_chain(certfile, keyfile, password)
```